pkgs/servers/inginious/default.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs, lib, pythonPackages }:
+with lib;
+
+let # XXX: Hack for PyLTI which supports only pytest_27 (for now).
+  pythonPackages = pkgs.python27Packages.override {
+    self = pythonPackages // { pytest = pythonPackages.pytest_27;};
+  };
+
+in let
+  docker_1_7_2 = pythonPackages.docker.override rec {
+    name = "docker-py-1.7.2";
+
+    src = pkgs.fetchurl {
+      url = "mirror://pypi/d/docker-py/${name}.tar.gz";
+      sha256 = "0k6hm3vmqh1d3wr9rryyif5n4rzvcffdlb1k4jvzp7g4996d3ccm";
+    };
+  };
+
+  webpy-custom = pythonPackages.web.override {
+    name = "web.py-INGI";
+    src = pkgs.fetchFromGitHub {
+      owner = "UCL-INGI";
+      repo = "webpy-INGI";
+      # tip of branch "ingi"
+      rev = "f487e78d65d6569eb70003e588d5c6ace54c384f";
+      sha256 = "159vwmb8554xk98rw380p3ah170r6gm861r1nqf2l452vvdfxscd";
+    };
+  };
+
+in pythonPackages.buildPythonApplication rec {
+  version = "0.3a2.dev0";
+  name = "inginious-${version}";
+
+  disabled = pythonPackages.isPy3k;
```

Comment:
Since this is tied to `python27Packages` wouldn't it be more appropriate to set `disabled = ! pythonPackages.isPy27`?